packages/flutter_test/lib/src/controller.dart

```diff
@@ -57,6 +67,26 @@ class SemanticsController {
 
   final WidgetsBinding _binding;
 
+  /// Gets the semantic root of the given [FlutterView].
+  ///
+  /// At the moment, all views share a semantic root, so all views will return
+  /// the same [SemanticsNode].
+  ///
+  /// See also:
+  ///
+  ///   * [root] for the semantic root of a single view application or
+  ///     widget test.
+  ///   * [WidgetTester.viewOf] to retrieve a specific view when multiple
+  ///     are available.
+  SemanticsNode rootOf({required FlutterView view}) {
+    // This will eventually look up the appropriate root for the given view.
+    // At the moment, all views share a root.
+    return _binding.pipelineOwner.semanticsOwner!.rootSemanticsNode!;
+  }
+
+  /// Gets the semantic root of a single view application or widget test.
+  SemanticsNode get root => rootOf(view: _viewFromPlatformDispatcher(_binding.platformDispatcher));
```

Comment:
Can we get away without this getter? if you really want this, you can always do `rootOf(tester.view)`, which is very explicit that you are gettin the root for a particular view and not for a 'random' one...

Reply:
I added it to be consistent with `WidgetTester`, where there was a `view` and `viewOf`. The thinking there being that users would likely just need to get "the" root, similarly to how `view` gets "the" view in most cases.

I don't technically need it for the work here, but I had hoped it'd help drive people away from using `tester.binding.pipelineOwner.semanticsOwner!.rootSemanticsNode!`, making that transition a bit easier.

Just from an aesthetics point of view, I like `tester.semantics.root` better than `tester.semantics.rootOf(view: tester.view)` as well. The latter, while explicit, is _very_ wordy.

I'm fine removing it, but there's my reasoning for it. Let me know what you'd like to do.

Reply:
I did end up making some logic in `simulatedAccessibilityTraversal` a bit cleaner with this getter, but still willing to remove it if you feel strongly about it.

Reply:
Removed per discussions over GVC.